transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollRegistration.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2018 The Netty Project
+ *
+ * The Netty Project licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.netty.channel.epoll;
+
+import io.netty.channel.unix.IovArray;
+
+import java.io.IOException;
+
+interface EpollRegistration {
```

Comment:
These would probably benefit from some docs.

Reply:
sure I can do... I did not bother as its package private anyway.